ANDES-TechReport-EN/02_methods.Rmd

```diff
@@ -47,8 +47,8 @@ The implementation of quality control checks in a data entry application is of p
 \begin{description}
 \item[Fishing sets]{The application should ensure all the required information about a fishing location have been filled in. Users should be warned if the fishing station start and/or end coordinates fall outside the expected sampling stratum (if applicable).}
 \item[Catches]{The application should verify the validity of catches that do not have any specimen-level data entry associated with them. This validation is important to help identify catches that might have been entered accidentally.}
-\item[Specimens]{The application should flag specimens whose length falls outside an acceptable range. Similarly, a validation of the specimen's length-to-weight ratio should be performed to warn users if a recorded weight falls outside the expected range for the measured length.}
-\item[Observations]{Individual observations are characterized by an observation type. Observation types should have predefined data types such as integer, float, string or categorical. The application should ensure that inputted observation values should never be left null and that they respect the data type of the corresponding observation type. In the case where an observation type (e.g., sex) has a set of defined categories (e.g., male, female, unknown), the application should ensure that any entered values fall within the set of available options.}
+\item[Specimens]{The application should flag specimens whose length falls outside an acceptable range. Similarly, a validation of the specimen's length-to-weight ratio should be performed to warn users if a recorded measurement falls outside the expected range.}
```

Comment:
I think there is QC check for both inputted length AND inputted weight. so I changed it to a general "measurement". 
I need to validate this...

Reply:
The length that is input is only subjected to "minimum length" and "maximum length" QC, so a flag is raised if the recorded length exceeds a species-specific threshold (e.g. a 150 cm Atlantic Cod would raise this flag). This is specified in the sampling requirements for each taxon
![image](https://github.com/dfo-gulf-science/ANDES-description-report-2023/assets/1110182/1d576e90-bd64-4334-9bf7-b2e74e6f597f)